R/lint.R

```diff
@@ -205,6 +194,43 @@ lint_package <- function(path = ".", relative_path = TRUE, ..., exclusions = NUL
 }
 
 
+#' Lint a package
+#'
+#' Apply one or more linters to all of the R files in a package.
+#' @param path the path to the base directory of the package, if \code{NULL},
+#' it will be searched in the parent directories of the current directory.
+#' @param relative_path if \code{TRUE}, file paths are printed using their path
+#' relative to the package base directory.  If \code{FALSE}, use the full
+#' absolute path.
+#' @param ... additional arguments passed to \code{\link{lint}}, e.g.
+#' \code{cache} or \code{linters}.
+#' @param exclusions exclusions for \code{\link{exclude}}, relative to the
+#' package path.
```

Comment:
The docs for `path`, `relative_path`, `...` and `exclusions` are almost identical to their definitions for `lint_dir`. You could use `@inheritParams   lint_package` to deduplicate the docs and rewrite the docs for `lint_dir` to apply generally for both `lint_dir` and `lint_package`

Reply:
hi @russHyde , thank you for the tip. I haven't used @inheritParams yet.